pillar/fluentd/init.sls

```diff
@@ -1,3 +1,10 @@
+{% set ENVIRONMENT = salt.grains.get('environment') %}
+{% set minion_id = salt.grains.get('id', '') %}
+{% set cert = salt.vault.cached_write('pki-intermediate-{}/issue/fluentd-client'.format(ENVIRONMENT), common_name='fluentd.{}.{}'.format(minion_id, ENVIRONMENT)) %}
```

Comment:
We should be passing the `cache_prefix` parameter with the minion_id for all of these cached_write calls